doc/quickstart/build.rst

```diff
@@ -69,7 +69,7 @@ Although CCF's unit tests can be run through ``ctest`` directly, the end-to-end
 
     .. code-block:: bash
 
-        $ TEST_ENCLAVE=virtual ./tests.sh -VV
+        $ TEST_ENCLAVE=virtual ./tests.sh [-VV]
```

Comment:
I think the bash code blocks are commands which could be pasted and run verbatim, so this shouldn't have the documenting-an-optional-arg brackets. If we think users don't want `-VV` all the time, we should just remove it (the comment above makes it clear that this is just a wrapper for `ctest`, so hopefully `--help` and `-VV` etc follow from there).